src/components/TypeDeleteWarningDialog/TypeDeleteWarningDialog.tsx

```diff
@@ -113,12 +105,14 @@ function TypeDeleteWarningDialog<T extends TypeBaseData>({
 
             <DashboardModal.Actions>
               {shouldShowViewAssignedItemsButton && (
-                <ConfirmButton
-                  onClick={() => navigate(viewAssignedItemsUrl)}
-                  transitionState="default"
+                <Link
+                  to={viewAssignedItemsUrl}
+                  style={{ pointerEvents: viewAssignedItemsUrl ? undefined : "none" }}
```

Comment:
question: Is it any sense to show this link when there is no `viewAssignedItemsUrl`?

Reply:
`viewAssignedItemsUrl` will always exist even though there may be no items assigned